editoast/src/infra.rs

```diff
@@ -125,6 +125,20 @@ impl Infra {
         }
     }
 
+    pub fn downgrade_generated_version(
+        &self,
+        conn: &PgConnection,
+    ) -> Result<Self, Box<dyn ApiError>> {
+        match update(osrd_infra_infra.filter(id.eq(self.id)))
+            .set(generated_version.eq(String::from("NULL")))
```

Comment:
Here the value will be  `"NULL"` (a string). What we want is the special psql value `NULL`.

```suggestion
            .set(generated_version.eq::<Option<String>>(None))
```